pkg/plugin/processor/builtin/impl/avro/decode.go

```diff
@@ -126,7 +127,7 @@ This processor is the counterpart to [` + "`avro.encode`" + `](/docs/processors/
 	}, nil
 }
 
-func (p *decodeProcessor) Configure(ctx context.Context, m map[string]string) error {
+func (p *decodeProcessor) Configure(ctx context.Context, m config.Config) error {
```

Comment:
I'd say that `m` makes less sense as a parameter name now that the type is `config.Config`

```suggestion
func (p *decodeProcessor) Configure(ctx context.Context, c config.Config) error {
```

Reply:
Changed everywhere 👍 You can save some time if you just tell me it's in all builtin processors, I'll find it, no worries 😄